lib/smart_app_launch/cors_token_exchange_test.rb

```diff
@@ -0,0 +1,28 @@
+module SMARTAppLaunch
+  class CORSTokenExchangeTest < Inferno::Test
+    title 'SMART Token Endpoint Enables Cross-Origin Resource Sharing (CORS)'
+    description %(
+      For requests from a client's registered origin(s), CORS configuration permits access to the token
+      endpoint. This test verifies that the token endpoint contains the appropriate CORS header in the
+      response.
+    )
+    id :smart_cors_token_exchange
+    optional
```

Comment:
This test doesn't need to be optional since it omits for the cases where it's not needed.